pages/proyectos-de-ley/index.js

```diff
@@ -109,77 +167,112 @@ export default function Bills({ bills, metadata }) {
-              authorship={authorship
-                .filter(author => author.authorship_type === 'AUTOR')
-                .map(
-                  ({
-                    congressperson: {
-                      congressperson_slug,
-                      id_name,
-                      id_first_surname,
-                      id_second_surname,
-                    },
-                  }) => ({
-                    slug: congressperson_slug,
-                    name: `${id_name} ${id_first_surname} ${id_second_surname}`,
-                  }),
-                )}
-              billId={id}
-              billTitle={title}
-              committeeName={last_committee ?? void 0}
-              publicationDate={presentation_date}
-              status={last_status ?? ''}
-              lastUpdate={last(tracking).date}
-              as={CUI.ListItem}
-            />
-          ),
-        )}
-      </CUI.List>
-      <Pagination numberOfPages={metadata.totalPages} active={1} />
+      {isBillsLoading ? (
+        <CUI.Box textAlign="center">
+          <CUI.Spinner color="primary" />
+        </CUI.Box>
+      ) : isBillsSuccess && bills.length ? (
+        <CUI.List spacing="4">
+          {bills.map(
+            ({
+              id,
+              last_status,
+              authorship,
+              title,
+              last_committee,
+              presentation_date,
+              tracking,
+            }) => (
+              <BillCard
+                key={id}
+                authorship={authorship
+                  .filter(author => author.authorship_type === 'AUTOR')
+                  .map(
+                    ({
+                      congressperson: {
+                        congressperson_slug,
+                        id_name,
+                        id_first_surname,
+                        id_second_surname,
+                      },
+                    }) => ({
+                      slug: congressperson_slug,
+                      name: `${id_name} ${id_first_surname} ${id_second_surname}`,
+                    }),
+                  )}
+                billId={id}
+                billTitle={title}
+                committeeName={last_committee ?? void 0}
+                publicationDate={presentation_date}
+                status={last_status ?? ''}
+                lastUpdate={last(tracking).date}
+                as={CUI.ListItem}
+              />
+            ),
+          )}
+        </CUI.List>
+      ) : isBillsSuccess && !bills.length ? (
+        <CUI.Text>No se encontraron resultados</CUI.Text>
+      ) : null}
+      {/* <Pagination numberOfPages={metadata.totalPages} active={1} /> */}
     </SidebarLayout>
   );
 }
 
-export const getStaticProps = () =>
-  fetch(`${process.env.api}bill`)
-    .then(data => data.json())
-    .then(data => ({ props: { bills: data.data, metadata: data } }))
-    .catch(error => ({ props: { error: error.toString() } }));
+const removeEmptyAttrFromObject = obj => {
```

Comment:
Me parece que si importáramos `omitBy` y también `isEmpty` de lodash podríamos conseguir el mismo resultado, también si podemos mover la función arriba del render. para que no se pierda.

```suggestion
const removeEmptyPropertiesFromObject = obj => omitBy(obj, isEmpty);
```